src/components/PeopleAndRoles/OrgPerson.vue

```diff
@@ -70,47 +70,58 @@
                 <!-- Roles -->
                 <template>
                   <label class="sub-header">Roles</label>
-                  <v-row>
-                    <v-col cols="4" v-if="isPerson">
-                      <div :class="{'highlightedRole': isRoleLocked(Roles.COMPLETING_PARTY)}">
+                  <v-row class="roles-row my-6">
```

Comment:
This row now has "gray" background.

Other spacing classes are to get everything nicely whitespaced in all cases.